src/main/resources/assets/catlas/lang/zh_TW.lang

```diff
@@ -7,6 +7,7 @@ catlas.config.map.toggle.show_player_names=顯示玩家名稱
 catlas.config.map.toggle.vanilla_head_marker=原版Minecraft地圖標記
 catlas.config.map.size.map_text_scale=地圖文字縮放
 catlas.config.map.size.player_heads_scale=玩家頭顱縮放
+catlas.config.map.size.head_border_scale=Head Border Scale
```

Comment:
If it's missing it should default to en_US already, so it doesn't need to be added here.